PresentScreenings/ListStreamer.cs

```diff
@@ -5,27 +5,38 @@
 namespace PresentScreenings.TableView
 {
 	/// <summary>
-    /// List reader, read a file and return is list of generic objects.
+    /// List streamer, read/write files based on list of derived objects.
     /// </summary>
 
-    public abstract class ListReader<T> where T : class, new()
+    public abstract class ListStreamer<T>  where T : ListStreamer<T>, new()
```

Comment:
Please consider to change ListStreamer as to no longer be generic.
The generic type is the same as the derived class in all cases.